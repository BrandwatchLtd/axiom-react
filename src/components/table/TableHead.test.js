import React from 'react';
import renderer from 'react-test-renderer';
import { TableHead, TableRow } from 'bw-axiom';

function getComponent() {
  return renderer.create(
    <TableHead>
      <TableRow />
    </TableHead>
  );
}

describe('TableHead', () => {
  it('renders', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
