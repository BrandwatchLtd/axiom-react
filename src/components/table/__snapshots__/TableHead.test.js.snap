// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHead renders 1`] = `
<thead
  className="ax-table__head"
>
  <tr
    className="ax-table__row"
  />
</thead>
`;
